images/tf-base/Dockerfile

```diff
@@ -1,18 +1,3 @@
-# Copyright 2019 Cortex Labs, Inc.
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#     http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-
-
 FROM ubuntu:18.04
```

Comment:
Did you consider using `FROM tensorflow/tensorflow:1.12.0-py3` here? Are there any advantages/disadvantages?

Reply:
don't need to, seems like tf-model-server doesnt need it